src/common/utils/image-utils.ts

```diff
@@ -0,0 +1,15 @@
+// TODO: Convert this functions into a middleware
+
+const ALLOWED_FILE_TYPES = ['image/png', 'image/jpeg', 'image/jpg', 'image/gif'];
+const ALLOWED_FILE_SIZE_IN_MB = 1000;
+
+export const isValidExtension = (image: Express.Multer.File) => {
+  if (!ALLOWED_FILE_TYPES.includes(image.mimetype)) return false;
```

Comment:
Only like suggest
`
... 
if (!ALLOWED_FILE_TYPES.includes(image.mimetype)) return false;
...
`
Boolean(!ALLOWED_FILE_TYPES.includes(image.mimetype))
`

`

Reply:
I like it